salt-mode.el

```diff
@@ -82,6 +82,50 @@ suitable for spellchecking."
 
 (put 'salt-mode 'flyspell-mode-predicate #'salt--flyspell-predicate)
 
+(defun salt--state-module-at-point ()
+  "Get the state module at point, either pkg or pkg.installed, or return nil."
```

Comment:
[Here's what I had for state-function at-point in my eldoc work](https://github.com/joewreschnig/salt-mode/blob/c8005d65234a7a4f0327fa939fcb489a8655ba32/salt-mode.el#L137).